Missing dependency to @angular/forms

Hi,

I just figured out (starting a new project) that there is a missing dependency when I build a Angular app with `@angular/material` without installing myself `@angular/forms`:

```
ERROR in node_modules/@angular/material/core/typings/error/error-options.d.ts(8,57): error TS2307: Cannot find module '@angular/forms'.
node_modules/@angular/material/core/typings/common-behaviors/error-state.d.ts(8,55): error TS2307: Cannot find module '@angular/forms'
```

Should we not add it as peer dependency ???

#### What is the expected behavior?
Install `@angular/forms` if needed by `@angular/material`.

#### What is the current behavior?
`@angular/forms` is not installed as dependency of `@angular/material`.

#### What are the steps to reproduce?

  * Start a new project (without the CLI schematics)
  * Install angular core packages and @angular/material
  * (Try to) build the application

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 7.2.3
Node: 10.13.0
OS: linux x64
Angular: 7.2.2
... animations, common, compiler, compiler-cli, core
... platform-browser, platform-browser-dynamic, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.12.3
@angular-devkit/build-angular     0.12.3
@angular-devkit/build-optimizer   0.12.3
@angular-devkit/build-webpack     0.12.3
@angular-devkit/core              7.2.3
@angular-devkit/schematics        7.2.3
@angular/cdk                      7.3.1
@angular/cli                      7.2.3
@angular/material                 7.3.1
@angular/pwa                      0.12.3
@angular/router                   7.2.3
@ngtools/webpack                  7.2.3
@schematics/angular               7.2.3
@schematics/update                0.12.3
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.28.4
```

#### Is there anything else we should know?
No

--- a/src/lib/package.json
+++ b/src/lib/package.json
@@ -25,7 +25,8 @@
     "@angular/animations": "0.0.0-NG",
     "@angular/cdk": "0.0.0-PLACEHOLDER",
     "@angular/core": "0.0.0-NG",
-    "@angular/common": "0.0.0-NG"
+    "@angular/common": "0.0.0-NG",
+    "@angular/forms": "0.0.0-NG"
   },
   "dependencies": {
     "tslib": "^1.7.1"

--- a/src/lib/schematics/ng-add/index.spec.ts
+++ b/src/lib/schematics/ng-add/index.spec.ts
@@ -48,6 +48,8 @@ describe('ng-add schematic', () => {
     expect(dependencies['@angular/material']).toBeDefined();
     expect(dependencies['@angular/cdk']).toBeDefined();
     expect(dependencies['hammerjs']).toBeDefined();
+    expect(dependencies['@angular/forms']).toBe(angularCoreVersion,
+      'Expected the @angular/forms package to have the same version as @angular/core.');
     expect(dependencies['@angular/animations']).toBe(angularCoreVersion,
       'Expected the @angular/animations package to have the same version as @angular/core.');
 

--- a/src/lib/schematics/ng-add/index.ts
+++ b/src/lib/schematics/ng-add/index.ts
@@ -25,13 +25,14 @@ export default function(options: Schema): Rule {
     // of the CLI project. This tag should be preferred because all Angular dependencies should
     // have the same version tag if possible.
     const ngCoreVersionTag = getPackageVersionFromPackageJson(host, '@angular/core');
+    const angularDependencyVersion =  ngCoreVersionTag || requiredAngularVersionRange;
 
     // In order to align the Material and CDK version with the other Angular dependencies,
     // we use tilde instead of caret. This is default for Angular dependencies in new CLI projects.
     addPackageToPackageJson(host, '@angular/cdk', `~${materialVersion}`);
     addPackageToPackageJson(host, '@angular/material', `~${materialVersion}`);
-    addPackageToPackageJson(host, '@angular/animations',
-        ngCoreVersionTag || requiredAngularVersionRange);
+    addPackageToPackageJson(host, '@angular/forms', angularDependencyVersion);
+    addPackageToPackageJson(host, '@angular/animations', angularDependencyVersion);
 
     if (options.gestures) {
       addPackageToPackageJson(host, 'hammerjs', hammerjsVersion);

